app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Comments/UnsubscribeEmail_spec.js

```diff
@@ -0,0 +1,12 @@
+/// <reference types="Cypress" />
+
+describe("Ubsubscribe comment email test spec", function() {
```

Comment:
```suggestion
describe("Unsubscribe comment email test spec", function() {
```
Super minor typo fix.